apps/frontend/src/libs/components/select/libs/types/option.type.ts

```diff
@@ -0,0 +1,6 @@
+type Option<T = string> = {
```

Comment:
This type will be used as a common type in different places in the application, so let's move it to `apps/frontend/src/libs/types/select-option.type.ts` and rename to SelectOption